### PR TITLE
Corrected gitpath of pipeline-scanner and added gitignore.

### PR DESCRIPTION
**Pull request by mohammedzee1000**
This is a correction to the pipe-scanner image git path as well as adding a gitignore to ignore everything except specific files.

### CODE REVIEW

**Comment by bamachrn** [comment]
@mohammedzee1000 gitignore ignores yml files in index.d. any reason for that?

**Comment by mohammedzee1000** [comment]
No, the first line makes it ignore everything and then the !pattern gives except that
Its like 

Deny all
Allow this

type of setup

File: index.d/pipeline-images.yml [hunk 1]
```diff
@@ -48,7 +48,7 @@
     job-id: pipeline-scanner
     git-url: git://github.com/dharmit/container-pipeline-service
     git-branch: master
-    git-path: pipeline_scanner
+    git-path: atomic_scanners/pipeline-scanner
```
Comment (kbsingh) [inline index.d/pipeline-images.yml]: Should this not point back to master in the centos/ pipeline service?

Comment (bamachrn) [inline index.d/pipeline-images.yml]: ideally yes, @dharmit was doing changes on this so thought may be we can build from his branch.

**Comment by mohammedzee1000** [comment]
Closing this, will send .gitignore in separate PR once https://github.com/CentOS/container-index/pull/33 and https://github.com/CentOS/container-index/pull/32 are merged

**Pull request closed without merging**